lib/cusparse/level2.jl

```diff
@@ -4,13 +4,13 @@
 export sv2!, sv2, mv!
 
 """
-    mv!(transa::SparseChar, alpha::BlasFloat, A::CuSparseMatrix, X::CuVector, beta::BlasFloat, Y::CuVector, index::SparseChar)
+    mv!(transa::SparseChar, alpha::Number, A::CuSparseMatrix, X::CuVector, beta::Number, Y::CuVector, index::SparseChar)
 
 Performs `Y = alpha * op(A) * X + beta * Y`, where `op` can be nothing (`transa = N`),
-tranpose (`transa = T`) or conjugate transpose (`transa = C`). `X` is a sparse vector, and
-`Y` is dense.
+tranpose (`transa = T`) or conjugate transpose (`transa = C`).
+`X` and `Y` are dense vectors.
 """
-mv!(transa::SparseChar, alpha::BlasFloat, A::CuSparseMatrix, X::CuVector, beta::BlasFloat, Y::CuVector, index::SparseChar)
```

Comment:
Which non-BlasFloat number types do we support?

Reply:
All number types that can be converted into `BlasFloat = eltype(A)` during the `ccall`.
We use
```julia
Ref{T}(alpha)
```
in the `ccall`.